field/checkbox/tests/behat/settings_configuration_04.feature

```diff
@@ -1,5 +1,5 @@
 @mod @mod_surveypro @surveyprofield @surveyprofield_checkbox
-Feature: Validate creation and submit for "checkbox" elements using the principal combinations of settings (4 of 4)
+Feature: validate creation and submit for "checkbox" elements using the principal combinations of settings (4 of 4)
```

Comment:
Not super-important, but it has called my attention that you are lower-casing all these feature names. I think that, in Moodle, we use to keep the first letter always upper, maybe because all the examples are that way, or because it's like a "literal that somebody says"... but the reality is that we uppercase that like a separate sentence....

Again, not critical al tall, just called my attention.

Reply:
I will fix this as soon I return in the office. Thanks.